server/node/src/main/java/org/kaaproject/kaa/server/admin/controller/UserController.java

```diff
@@ -289,6 +290,26 @@ public void deleteUser(
        return  userService.findAllTenantAdminsByTenantId(tenantId);
     }
 
+    @ApiOperation(value = "Get endpoint user configuration by external user id",
+            notes="Get endpoint user configuration by external user id. Only user with TENANT_DEVELOPER and TENANT_USER roles is allowed to perform this operation.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "The specified url   is not valid"),
+            @ApiResponse(code = 401, message = "The user is not authenticated or invalid credentials were provided"),
+            @ApiResponse(code = 403, message = "The authenticated user does not have neither TENANT_DEVELOPER nor TENANT_USER role"),
+            @ApiResponse(code = 500, message = "An unexpected error occurred on the server side")})
+    @RequestMapping(value = "configuration/{appToken}/{userId}/{schemaVersion}", method = RequestMethod.GET)
```

Comment:
@RequestMapping(value = "configuration/{userId}/{appToken}/{schemaVersion}", method = RequestMethod.GET) is more accurate